Readme.md

```diff
@@ -1,39 +1,66 @@
-<!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128533541/16.1.4%2B)
-[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T150957)
-[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
-<!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
-
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView- How to conditionally enable and disable the batch editing on the client side
+# Grid View for ASP.NET Web Forms - How to control data editing on the client side in batch edit mode
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t150957/)**
 <!-- run online end -->
 
+This example demonstrates how to use the check box state to control data editing in the grid in batch edit mode.
+
+![Allow Edit Checkbox](controlEditing.gif)
+
+## Overview
+
+Follow the steps below:
+
+1. Create the [Grid View](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridView) control and populate it with columns. Set the grid's [SettingsEditing.Mode](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxGridViewEditingSettings.Mode) property to `Batch` to enable the batch edit mode. Add a [GridViewCommandColumn](https://docs.devexpress.devx/AspNet/DevExpress.Web.GridViewCommandColumn) and set its [ShowNewButtonInHeader](https://docs.devexpress.devx/AspNet/DevExpress.Web.GridViewCommandColumn.ShowNewButtonInHeader) and [ShowDeleteButton](https://docs.devexpress.devx/AspNet/DevExpress.Web.GridViewCommandColumn.ShowDeleteButton) properties to `true`.
+
+    ```aspx
+    <dx:ASPxGridView ID="ASPxGridView1" runat="server" KeyFieldName="ID" ...>
+        <SettingsEditing Mode="Batch" />
+        <Columns>
+            <dx:GridViewCommandColumn ShowNewButtonInHeader="true" ShowDeleteButton="true" />
+            <!-- ... -->
+        </Columns>
+    </dx:ASPxGridView>
+    ```
 
-<p>This example demonstrates how to conditionally enable or disable the editing on the client side.</p>
-<p><strong>See also:</strong><br><a href="https://www.devexpress.com/Support/Center/p/T150965">GridView - Batch Editing - How to conditionally enable and disable the editing on the client side</a></p>
+2. Add a check box and handle its client-side [CheckedChanged](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxCheckBox.CheckedChanged) event. In the handler, create a flag variable, get the current state of the check box, and assign this value to the flag variable.
```

Comment:
> In the handler, create a flag variable, get the current **state** of the check box, and assign this **value** to the flag variable.

Implied synonyms. You used "value" as a synonym of "state" but you never introduced them as such. Since you've said "get the state", it already implies that it is a value. Therefore, we can refer to this value as a "state" in the "assign" context:

```suggestion
2. Add a check box and handle its client-side [CheckedChanged](https://docs.devexpress.devx/AspNet/DevExpress.Web.ASPxCheckBox.CheckedChanged) event. In the handler, create a flag variable, get the current state of the check box, and assign the state to the flag variable.
```